generateapis.sh

```diff
@@ -40,21 +46,26 @@ fetch_github_repos() {
     git clone --recursive https://github.com/googleapis/gapic-generator-csharp
   fi
 
-  if [ -d "googleapis" ]
+  if [[ "$SYNTHTOOL_GOOGLEAPIS" != "" ]]
```

Comment:
I'm not very familiar with the context of the PR, but I was expecting this to simply replace "googleapis" with $GOOGLEAPIS ?

Reply:
OK, so shouldn't it be `==`, not `!=`?

And what's going on - you're meant to be on vacation??

Reply:
Was just idly checking emails, but won't look at detail of PR now :)

Reply:
Whoops - yes, that should indeed be ==. Fixed, and removed the echo lower down.